ContentPatcher/docs/i18n/zh/token-strings-api.md

```diff
@@ -0,0 +1,192 @@
+﻿← [README](README.md)
+
+此文档帮助SMAPI模组作者在自己的模组中使用Content Patcher的令牌字符串。
+
+**若需要添加给其他内容包使用的自定义令牌，请参阅[扩展性API](extensibility.md)。其他信息请参见[主README](README.md)**
+
+**🌐 其他语言： [en (English)](../../token-strings-api.md)。**
+
+## 目录
+* [概述](#overview)
+* [访问API](#access-the-api)
+* [解析令牌字符串](#parse-token-strings)
+* [管理令牌字符串](#manage-token-strings)
+* [注意事项](#caveats)
+* [参见](#see-also)
+
+## 概述<a name="overview"></a>
+Content Patcher有一个令牌系统(author-guide/tokens.md)，允许内容包组成复杂并遵守上下文的字符串。
+```js
+""
+"My favorite season is {{Season}}." // If a save is loaded, {{Season}} will be replaced with the current season.
```

Comment:
We should probably translate this code example too (except for the actual `{{Season}}` token).